src/rlgl.h

```diff
@@ -548,6 +548,7 @@ RLAPI void rlLoadMesh(Mesh *mesh, bool dynamic);                          // Upl
 RLAPI void rlUpdateMesh(Mesh mesh, int buffer, int count);                // Update vertex or index data on GPU (upload new data to one buffer)
 RLAPI void rlUpdateMeshAt(Mesh mesh, int buffer, int count, int index);   // Update vertex or index data on GPU, at index
 RLAPI void rlDrawMesh(Mesh mesh, Material material, Matrix transform);    // Draw a 3d mesh with material and transform
+RLAPI void rlDrawMeshInstanced(Mesh mesh, Material material, int count, Matrix *transforms);    // Draw a 3d mesh with material and transform
```

Comment:
I like this function, just proposing a change in signature: 
```c
RLAPI void rlDrawMeshInstanced(Mesh mesh, Material material, Matrix *transforms, int count);
```

Reply:
Done